xandikos/access.py

```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```

Comment:
I added the import there with a comment. Also deleted the remaining imports and nothing crashed this time. 🌮